zigzag/data/configs/zigzag-config-example.json

```diff
@@ -0,0 +1,5 @@
+{
+    "test_cycle": "test_cycle_name",
+    "module_hierarchy": "{{ MODULE_HIERARCHY }}",
+    "path_to_test_exec_dir": "{{ PATH_TO_TEST_EXEC_DIR }}"
```

Comment:
It been too long for me to remember if we have answered this already but .....
Is there a way to reference properties in the XML?  Things that may have been captured by an earlier tool?  (pytest-zigzag)

Reply:
For example I saw this in the pytest-zigzag PR 
`"MODULE_HIERARCHY": {"type": ["array", "null"] },`
How would we use that value here in module_hierarchy?

Reply:
I need to write this up in the docs, but yeah, you can reference any property you've specified in the xml.  So, to plumb it though, you can add an arbitrary_value to the config for pytest-zigzag, it will become a property in the resulting junit.xml, and you can interpolate it into the zigzag config by adding {{ arbitrary_value }}.

Reply:
What does the `path_to_test_exec_dir` do exactly?

Reply:
This is to handle the issue that @zreichert ran into with mk8s.  If a repo has tests in a nested dir, and they are executed from there, it's tough to put together a failure link to the problematic line in the repo without knowledge about the intervening path. The path_to_test_exec_dir property just stores that info if it's relevant.  If it's not defined, it should be ignored.